### DISCUSSION TITLE
Addition of Nyx to docker image

### DISCUSSION THREAD

**Discussion by Heppie** [Ilshidur/tor-relay-docker#28] [Ideas]
Having Nyx as part of the docker to enable quick monitoring of node status

**Comment by arhue** [comment]
There's also this https://github.com/ralphwetzel/theonionbox

**Comment by adriadam10** [comment]
> Having Nyx as part of the docker to enable quick monitoring of node status

The owner doesn't want to add nyx to the image:
[https://github.com/Ilshidur/tor-relay-docker/pull/22#issuecomment-1732311368](url)

I have a forked version with it: [https://github.com/adriadam10/tor-relay-docker-nyx](url)

**Comment by Ilshidur** [comment]
Hello, and happy new year !

Indeed, I think it's best to keep this image as lightweight as possible for now. From what I can see, Nyx can be configured to remotely monitor a container based on this image using the exposed control port. Just make sure to have an authentication mechanism protecting this port.

You can configure the container for this purpose by setting these env vars [(see README)](https://github.com/Ilshidur/tor-relay-docker#configuration) :
```
TOR_ControlPort=9051
TOR_CookieAuthentication=1
```

Converting the issue to a discussion...
